server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -191,6 +191,7 @@ public class PostgresWireProtocol {
 
     public static int SERVER_VERSION_NUM = 100500;
     public static String PG_SERVER_VERSION = "10.5";
+    public static String DATE_STYLE = "ISO";
```

Comment:
I would consider this as a single source of truth in line with the server versions.

Adding it to `SessionSettings` / `SessionContext` would lead to quite some changes where sessions settings are used today. If implemented as immutable value, this would lead to rework, when we would decide to implement a changeable date style.

Reply:
outdated / is now using the session setting